helm/Chart.yaml

```diff
@@ -1,9 +1,4 @@
-apiVersion: v2 # helm 3
-type: application
-name: permit-backend-v2
-description: A Helm chart for Permit.io Backend service
-version: 0.0.1
-dependencies:
-  - name: permit-io-chart
-    version: ^0.17.13
-    repository: "oci://076477451822.dkr.ecr.us-east-2.amazonaws.com"
+apiVersion: v2
+name: pdp
+description: A Helm chart for deploying pdp application
```

Comment:
Would change to "An official Helm chart for Permit.io PDP (Policy Decision Point)"